galley/pkg/server/configmap.go

```diff
@@ -19,7 +19,7 @@ import (
 	"io/ioutil"
 	"sync"
 
-	"github.com/howeyc/fsnotify"
```

Comment:
i'm guessing that we use this elsewhere since it is not deleted from the vendor directory

Reply:
Yes, we should switch everything else over to `github.com/fsnotify/fsnotify` eventually. I made a one-off change for this PR because it wasn't possible to mock the `github.com/howeyc/fsnotify` watcher and event.